src/components.ts

```diff
@@ -0,0 +1,47 @@
+import path from "path"
+import upperFirst from "lodash.upperfirst"
+import camelCase from "lodash.camelcase"
+import glob from "glob"
+import fs from "fs-extra"
+import svgr from "@svgr/core"
+
+import { Debugger } from "./Debugger"
+
+import { AssetsConfig } from "./index"
+
+const svgrOptions = {
+  dimensions: false, // we want direct control over dimensions
+  icon: true, // preserve viewBox property
+  replaceAttrValues: {
+    // map all relevant fills to currentColor
+    "#222": "currentColor",
+    "#232A36": "currentColor",
+    "#323232": "currentColor",
+    "#F73B47": "currentColor",
+    "#FF301C": "currentColor",
+    "#FFFFFF": "currentColor",
```

Comment:
🤔 we should probably handle this in the source files. we have so many different colors that I don't think it's feasible to list them all here. I would not want to see PRs just to list another color when adding a new icon - coupling is off

Reply:
I'm ok taking this as a smell for the time being, but I'd like to see what other options we have. can you decide whether to create a tech20 or tackle it with the implementation?

Reply:
this part was on my todo list for today. I need a way to configure this somehow because while DH has the same config as listing, we might want to have it not shared across all projects

Reply:
extracted to config